pom.xml

```diff
@@ -48,4 +81,77 @@
       </dependency>
     </dependencies>
   </dependencyManagement>
+
+  <profiles>
+    <profile>
+      <id>release</id>
+
+      <distributionManagement>
+        <repository>
+          <id>bintray</id>
+          <url>https://api.bintray.com/maven/opentracing/maven/opentracing-java/;publish=1</url>
```

Comment:
thanks :+1: !! Yes it should be the package name. https://bintray.com/docs/api/#_maven_upload